src/parser/jobs/gnb/modules/NoMercy.tsx

```diff
@@ -37,228 +34,87 @@ const EXPECTED_USES = {
-			content: <Trans id="gnb.nomercy.suggestions.expected-uses.content">
-				Watch your uses of certain abilities during <ActionLink {...ACTIONS.NO_MERCY}/>. Under ideal conditions, you should
-				be using <ActionLink {...ACTIONS.SONIC_BREAK}/>, a full <ActionLink {...ACTIONS.GNASHING_FANG}/> combo, and all of
-				your off-GCD skills - <ActionLink {...ACTIONS.BLASTING_ZONE}/>, <ActionLink {...ACTIONS.BOW_SHOCK}/>, and at least one
-				charge of <ActionLink {...ACTIONS.ROUGH_DIVIDE}/> - under the buff duration.
-			</Trans>,
-			why: <Trans id="gnb.nomercy.suggestions.expected-uses.why">
-				<Plural value={sumMissingExpectedUses} one="# expected cast" other="# expected casts"/> missed during <StatusLink {...STATUSES.NO_MERCY}/> windows.
-			</Trans>,
-			tiers: SEVERITIES.MISSING_EXPECTED_USES,
-			value: sumMissingExpectedUses,
-		}))
+		return 0
 	}
 
-	output() {
-		return <RotationTable
-			targets={[
-				{
-					header: <Trans id="gnb.nomercy.table.header.gcds">GCDs</Trans>,
-					accessor: 'gcds',
-				},
-				{
-					header: <ActionLink showName={false} {...ACTIONS.BLASTING_ZONE}/>,
-					accessor: 'blastingZone',
-				},
-				{
-					header: <ActionLink showName={false} {...ACTIONS.SONIC_BREAK}/>,
-					accessor: 'sonicBreak',
-				},
-				{
-					header: <ActionLink showName={false} {...ACTIONS.ROUGH_DIVIDE}/>,
-					accessor: 'roughDivide',
-				},
-				{
-					header: <ActionLink showName={false} {...ACTIONS.BOW_SHOCK}/>,
-					accessor: 'bowShock',
-				},
-				{
-					header: <ActionLink showName={false} {...ACTIONS.GNASHING_FANG}/>,
-					accessor: 'gnashingFang',
-				},
-			]}
-			data={this.noMercyWindows
-				.map(window => {
-					return ({
-						start: window.start - this.parser.fight.start_time,
-						end: window.end != null ?
-							window.end - this.parser.fight.start_time
-							: window.start - this.parser.fight.start_time,
-						targetsData: {
-							gcds: {
-								actual: window.numGcds,
-								expected: EXPECTED_USES.GCD,
-							},
-							blastingZone: {
-								actual: window.numBlastingZones,
-								expected: EXPECTED_USES.BLASTING_ZONE,
-							},
-							sonicBreak: {
-								actual: window.numSonicBreaks,
-								expected: EXPECTED_USES.BOW_SHOCK,
-							},
-							roughDivide: {
-								actual: window.numRoughDivides,
-								expected: EXPECTED_USES.ROUGH_DIVIDE,
-							},
-							bowShock: {
-								actual: window.numBowShocks,
-								expected: EXPECTED_USES.BOW_SHOCK,
-							},
-							gnashingFang: {
-								actual: window.numGnashingFangs,
-								expected: EXPECTED_USES.GNASHING_FANG,
-							},
-						},
-						rotation: window.rotation,
-					})
-				})
-			}
-			onGoto={this.timeline.show}
-		/>
+	getBuffWindowNotes(buffWindow: BuffWindowState) {
+		return buffWindow.rotation.find(cast => cast.ability.guid === ACTIONS.BLOODFEST.id) ?
+			<Trans id="gnb.nomercy.chart.notes.yes"> Yes </Trans> : <Trans id = "gnb.nomercy.chart.notes.no"> No </Trans>
```

Comment:
Again what's up with the spacing? I wonder if we could consider using a tick/cross or something but then localising would be weird since idk how crowdin would handle unicode emojis (JP needs a circle instead of tick).

Reply:
At the moment we've got some jobs using green checks and red x's. and others just using Yes and No. some uniformity might be useful

Reply:
We prob should really standardise this, there's a few other things that are kinda mix and match (like some jobs calling stuff using buff window as "$thing Windows" and others just "$thing".